src/main/java/uk/gov/hmcts/reform/sscs/personalisation/Personalisation.java

```diff
@@ -294,17 +300,17 @@ public Template getTemplate(E notificationWrapper, Benefit benefit, Subscription
     private String getTemplateConfig(SubscriptionType subscriptionType,
                                      NotificationEventType notificationEventType) {
         String templateConfig = notificationEventType.getId();
-        if (APPEAL_LAPSED_NOTIFICATION.equals(notificationEventType)
-            || APPEAL_WITHDRAWN_NOTIFICATION.equals(notificationEventType)
-            || EVIDENCE_RECEIVED_NOTIFICATION.equals(notificationEventType)
-            || SYA_APPEAL_CREATED_NOTIFICATION.equals(notificationEventType)
-            || RESEND_APPEAL_CREATED_NOTIFICATION.equals(notificationEventType)
+        if (ADJOURNED_NOTIFICATION.equals(notificationEventType)
             || APPEAL_DORMANT_NOTIFICATION.equals(notificationEventType)
-            || ADJOURNED_NOTIFICATION.equals(notificationEventType)
```

Comment:
Why has ADJOURNED_NOTIFICATION been removed?

Reply:
I alphabetised these because it hurts my brain to read them individually instead of just skipping to the one I want. ADJOURNED_NOTIFICATION is up on line 303 now.

Reply:
I still can't see ADJOURNED_NOTIFICATION, please check the whole section!

Reply:
It's back after I resolved the conflict again.